File: MyMindNotes/app/src/main/java/com/android/mymindnotes/retrofit/DeleteUserApi.java
package com.android.mymindnotes.retrofit;

import java.util.Map;

import retrofit2.Call;
import retrofit2.http.DELETE;
import retrofit2.http.Path;

public interface DeleteUserApi {
    @DELETE("/api/member/delete/{user_index}")
    Call<Map<String, Object>> deleteUser(@Path("user_index") int user_index);
}

File: MyMindNotes/data/src/main/java/com/android/mymindnotes/data/retrofit/api/user/DeleteUserApi.kt
package com.android.mymindnotes.data.retrofit.api.user

import com.android.mymindnotes.core.dto.DeleteUserResponse
import retrofit2.http.DELETE
import retrofit2.http.Path

interface DeleteUserApi {
    @DELETE("/api/member/delete/{user_index}")
    suspend fun deleteUser(
        @Path("user_index") user_index: Int
    ): DeleteUserResponse
}